rust_snuba/rust_arroyo/Cargo.toml

```diff
@@ -12,9 +12,9 @@ env_logger = "0.9.0"
 futures = "0.3.21"
 log = "0.4.17"
 once_cell = "1.18.0"
-rdkafka = { version = "0.36", features = ["cmake-build"] }
+rdkafka = { version = "0.36", default-features = false, features = ["cmake-build"] }
```

Comment:
the reason for this is that `rdkafka` has a defauft feature for zlib compression, which for reasons I do not understand caused some linker problems locally for me